Detectors/MUON/MID/Raw/include/MIDRaw/LinkDecoder.h

```diff
@@ -45,10 +46,12 @@ class GBTDecoder
   std::function<void(gsl::span<const uint8_t>, uint32_t orbit, std::vector<ROBoard>& data, std::vector<ROFRecord>& rofs)> mDecode{nullptr};
 };
 
-std::unique_ptr<GBTDecoder> createGBTDecoder(const o2::header::RDHAny& rdh, uint16_t feeId, bool isDebugMode, uint8_t mask, const ElectronicsDelay& electronicsDelay);
-std::unique_ptr<GBTDecoder> createGBTDecoder(uint16_t feeId, bool isBare = false, bool isDebugMode = false, uint8_t mask = 0xFF, const ElectronicsDelay& electronicsDelay = ElectronicsDelay());
+std::unique_ptr<LinkDecoder> createGBTDecoder(const o2::header::RDHAny& rdh, uint16_t feeId, bool isDebugMode, uint8_t mask, const ElectronicsDelay& electronicsDelay);
```

Comment:
Each time I encounter GBT and Link methods side by side I'm getting confused. Would probably be good to insist in the README that GBT==Link in bare mode and that Link={GBTs} in UL mode (hope I got it correct :wink:)

Reply:
Yes, you got it right. The point is that until the latest UL version, the data from each GBT link was streamed independently, with its own RDH. So there was a unique relation between FEEId, GBT Link ID and LinkId in the RDH.
With the new UL this correspondence is no longer there. The FEEId is associated to the UL link, which collects the input of several GBT Ids.
I changed the name in the more meaningful createLinkDecoder everywhere, but here, because the RawGBTDecoderSpec is supposed to do what its name says: i.e. decode a GBT link. Of course, this will not be used anymore for UL Data....